js/front/dataset/resource-modal.vue

```diff
@@ -7,6 +7,8 @@
 
         <dl class="dl-horizontal dl-wide">
           <dt>{{ _('URL') }}</dt>
+          <dd><a :href="resource.contentUrl" @click="onClick">{{resource.contentUrl}}</a></dd>
```

Comment:
Isn't it `.url` here and `.permalink` below? Where does that `contentUrl` come from?